src/main.rs

```diff
@@ -145,8 +150,11 @@ impl EventHandler for HandlerWrapper {
 }
 
 async fn build_handler() -> anyhow::Result<Handler> {
+
+    let lp = lp::LP::new();
+
     let conn = Connection::open("humble_ledger.sqlite")?;
-    let polls = ModPoll::new("✅", "❎", "▶️", None, "<a:crabrave:996854529742094417>");
+    let polls = ModPoll::new("✅", "❎", "▶️", None, "<a:crabrave:996854529742094417>", lp.clone() );
```

Comment:
I wonder if we could implement some kind of event system in Handler, instead of explicitly connecting modules.

Reply:
Yes, this is the big ugly. I just hacked together the dumbest thing I could think off to make it work.

Reply:
I've implemented this now, and it relies on https://github.com/etwyniel/discord_framework/pull/2